bse/bsebus.cc

```diff
@@ -1014,13 +1014,13 @@ BusImpl::right_volume (double val)
 bool
 BusImpl::master_output() const
 {
-  BseBus *self = const_cast<BusImpl*> (this)->as<BseBus*>();
-  auto parent = BSE_ITEM (self)->parent;
+  BseBus  *self = const_cast<BusImpl*> (this)->as<BseBus*>();
+  BseItem *parent = BSE_ITEM (self)->parent;
   if (BSE_IS_SONG (parent))
     {
       BseSong *song = BSE_SONG (parent);
       BseBus *master = bse_song_find_master (song);
-      return (self == master);
+      return self == master;
```

Comment:
Code like this can be merged down into the culprit of your current branch. Here's how you do that:

1) Use `git blame HEAD -- bsebus.cc` to find the faulty commit:
```
aaabbbccc   bse/bsebus.c swesterfeld     return (self == master);
```
2) Commit the fix as in:
```
git commit -m '+++ fix commit aaabbbccc' 
111222333
```
3) Assuming your current branch started from master, merge this down with: `git rebase -i master`
4) Move `pick 111222333 +++ fix commit aaabbbccc` into a `squash` or `fixup` following the culprit:
```
pick aaabbbccc BSE: introduce buggy code
squash 111222333 +++ fix commit aaabbbccc
```

Reply:
Thanks for the hint, I normally don't need this while coding, but I tried to fix everything here that could be done by history rewriting using git rebase -i now.